### ISSUE TITLE
UI Bug

### ISSUE THREAD

**Issue by creeper451** [hardcore-sushi/DroidFS#121]
its not a big issue but i figured i should report
[recording_20220423_142506.zip](https://github.com/hardcore-sushi/DroidFS/files/8546852/recording_20220423_142506.zip)

**Comment by hardcore-sushi** [comment]
It should have been fixed with ec348383c645c5eac37584b2a1ff867d941001c8. I built a test APK, can you test it and confirm that the issue has been solved?
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Download link: https://drop.underworld.fr/r/1Eqpbr4ERG#nI62BS7LxrTBQP5zmerPjgDNorueE0I56wNUrJ1+G0w=
SHA-256: bbddb75e37b1c14ead3c106c86dce298ad3d037f3cd28996cd7b66edd3003ce1
-----BEGIN PGP SIGNATURE-----

iHUEARYIAB0WIQS2Tv6GzuHQVPCCFxGv44Q0SkXhOgUCYm6iEgAKCRCv44Q0SkXh
Olh5APsECNrQ6YEpi56ZL3RjYZWDs5g7IpsWXE3tvFLyZ9D1LwEAnsklkIY//6RH
3SzU0vW3Gpflvf/JcrekwzsVIsoSAg0=
=wag6
-----END PGP SIGNATURE-----
```

**Comment by creeper451** [comment]
nope still broken

**Comment by hardcore-sushi** [comment]
OK it really should be fixed now: 5ea0b8ad410fca8b11608ecd4c383c73a84730be. Can you confirm please?
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Download link: https://drop.underworld.fr/r/fR2pmtQWjF#uzmxvhZI0Bodj6wb62K5+bCeNUDPOb3qmAJ84axSehc=
SHA-256: 319d380da7f9c06ecb0a8f9f68ef924d701d9c239ec1f910ec874a6bb778fc42

-----BEGIN PGP SIGNATURE-----

iHUEARYIAB0WIQS2Tv6GzuHQVPCCFxGv44Q0SkXhOgUCYm7JfAAKCRCv44Q0SkXh
OkFUAP9nin9DhSU+P+oTysh5zxL3uyF0IWWoFVVMWYtRkGOzjAD/fwVQZNz9TnvQ
VA1WO8536tBqwurklAiIW2Lq9f2RIgo=
=t4Me
-----END PGP SIGNATURE-----
```

**Comment by creeper451** [comment]
yes it worked this time
thank you
